setup.py

```diff
@@ -147,7 +147,9 @@ def get_extensions():
     )
 
     is_rocm_pytorch = False
-    if torch.__version__ >= '1.5':
+    TORCH_MAJOR = int(torch.__version__.split('.')[0])
+    TORCH_MINOR = int(torch.__version__.split('.')[1])
+    if TORCH_MAJOR > 1 or (TORCH_MAJOR == 1 and TORCH_MINOR >= 5):
```

Comment:
Should we be [using the newly introduced `TorchVersion` instead](https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/torch/torch_version.py#L13)?

Reply:
Created an issue to track this down https://github.com/pytorch/vision/issues/4296

Reply:
If we want to use the new [`torch.__version__` as a `TorchVersion` object](https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/torch/torch_version.py#L13) , then just reverting this pr will do the job.
Or can I create a new pr for just one line?

Reply:
>  Or can I create a new pr for just one line?

Yup :)

Reply:
Done.
Thanks Sir